pandas/tests/test_strings.py

```diff
@@ -232,10 +230,8 @@ def test_api_per_dtype(self, box, dtype, any_skipna_inferred_dtype):
             assert isinstance(t.str, strings.StringMethods)
         else:
             # GH 9184, GH 23011, GH 23163
-            with pytest.raises(
-                AttributeError,
-                match="Can only use .str " "accessor with string values.*",
-            ):
```

Comment:
with the black code style applied, I'm now beginning to prefer this layout rather than having the `msg=` on the previous line.

we currently did this where line length was an issue.